Project1_tr

```diff
@@ -0,0 +1,350 @@
+
+B ≡ M ⊓ ∃p2¯.Person
+
+(b)  A that says that x is an aunt of y
+
+A ≡ ¬M ⊓ (∃p2¯.(parent_of≥ 2. (∃parent_of. Person)) ⊔ ¬∃parent_of.Person)
+
+(c)  C that says that x and y are cousins
+
+gp2 (Grandparent with at least two children-parents) ≡ parentof≥2. (∃parentof.Person)
+  
+C ≡ gp2¯.Person
+  
+(d)  O that says that x is an only child  
+
+parent_only (parent of exactly one child) ≡ (parent_of.Person≥1 ) ⊓ (parent_of.Person≤1)
+
+O ≡ ∃parent_only¯.Person
+
+(e)  T that says that x has exactly two brothers
+
+p3m (parent of exactly three male children) ≡ (parent_of≥3.Male) ⊓ (parent_of≥3. Male) 
+
+T ≡ ∃p3m¯.Person
+
+
+5. Select two formulas defined in ALC from question 4 to form the basis of a T-Box. Supplement this T-box with whatever other axioms you like, 
+as well as an A-box, so that you ultimately construct a knowledge base K = (T,A). Provide a model of K. This may be graphical or symbolic or both.
+
+K = (T, A)      
+
+(a)  B that says that [w] is a brother of [x]
+(b)  A that says that [y] is an aunt of [w]
+
+T.box= {Brother ⊆ Male    (TBox.a)
+	Aunt ⊆ ~Male}	  (TBox.b)
+
+A.box=  {John: Bro ∩ male            (Ax1)
+         Paul: Bro ∩ male            (Ax2)
+         Jane: Aunt ∩ ~Male          (Ax3) 
+         (Joan, John): parent_of     (Ax4)
+         (Joan, Paul): parent_of     (Ax5)
+         (Jane, John): aunt_of}      (Ax6)
+            
+ΔI = {w, x, y, z}
+
+John = w
+Paul = x
+Jane = y
+Joan = z
+
+Male = {w, x}
+Female/Other non-male: {y,z}
+
+Brother = {w, x}
+Aunt = {y}
+Nephew = {w}
+Parent_of = {(z,w), (z,x)}
+Aunt_of = {(y,w), (y,x)}  
+
+
+  
+6. Explain the difference - using natural language - between the first-order prefixes:
+
+(a) ∃x∀y and ∀x∃y
+
+  ∃x∀y means “there is--or exists--some x such that for all y…”, and ∀x∃y means “For all x, there is some y that…”
+
+(b) ∃x∀y∃z and ∀x∃y∀z 
+
+  ∃x∀y∃z means “there is an x for all y, there is some z such that…”, whereas ∀x∃y∀z means “for all x there is some y, so that for all z…” 
+
+(c) ∀x∃y∀z∃w and ∃x∀y∃z∀w
+
+  ∀x∃y∀z∃w means “for all x there is some y for all z, there exists some w such that…”,  and ∃x∀y∃z∀w means “there is some x such that for all y, 
+  there is some z such that for all w…”
+
+
+
+7. Show that the following sentences are not equivalent by exhibiting a graph that models one but not both of these sentences:
+
+∀x∃y∀z(R(x,y) ∧ R(x,z) ∧ R(y,z))
+∃x∀y∃z(R(x,y) ∧ R(x,z) ∧ R(y,z))
+
+
+graph LR 
+
+  0 ----> 1 ----> 2
+   `--------------^
+  
+  U: {0,1,2}
+  R: {(0,1),(0,2),(1,2)}
+  
+  
+  
+8. Using an online tableau proof generator - such as the one found here https://www.umsu.de/trees/ - provide tree proofs of the following entailments, 
+which are known as the De Morgan's laws:
+
+(a) ∀x∀y(¬(Px ∧ Qx) → (¬Px ∨ ¬Qx))
+
+<img src="blob:chrome-untrusted://media-app/55dfa8c9-4008-4723-9c14-e4c8b00119b0" alt="Project1_8a.png"/>
```

Comment:
What justifies moving from the negated universal in line 1 to the negated universal in line 2?